package.json

```diff
@@ -47,27 +47,28 @@
   },
   "homepage": "https://github.com/ipfs-shipyard/ipfs-desktop",
   "devDependencies": {
-    "chai": "^4.3.4",
+    "@playwright/test": "^1.18.1",
+    "chai": "^4.3.6",
     "cross-env": "^7.0.3",
     "delay": "^5.0.0",
     "dirty-chai": "^2.0.1",
     "dotenv": "^14.2.0",
-    "electron": "13.6.7",
+    "electron": "^17.0.0",
```

Comment:
![cats-in-shower-caps19](https://user-images.githubusercontent.com/157609/152456590-a103a9ff-2df8-49da-a439-2b847017423b.jpg)